docs/interactions/Message_Components.md

```diff
@@ -300,6 +303,13 @@ The payloads for the select menu components are detailed in the [select menu str
 | emoji?       | partial [emoji](#DOCS_RESOURCES_EMOJI/emoji-object) object | `id`, `name`, and `animated`                             |
 | default?     | boolean                                                    | Will show this option as selected by default             |
 
+###### Select Default Value Structure
+
+| Field | Type      | Description                                                                   |
+|-------|-----------|-------------------------------------------------------------------------------|
+| id    | snowflake | ID of a user, role, or channel                                                |
+| type  | string    | Type of value that `id` represents. Either `"user"`, `"role"`, or `"channel"` |
```

Comment:
Are there plans for select menu default values to support the "default" property like select menu options?

Reply:
what would that be for? presence in default_values makes it a default

Reply:
Oh I thought default values were like choices but for non string type selects.
In that case, is validation for the min/max_values (going to be) enforced on default values as well?

Reply:
>  is validation for the min/max_values and proper types (going to be) enforced on default values as well
I can confirm from my testing that those validations are actually enforced

Reply:
> I can confirm from my testing that those validations are actually enforced


Would be worth noting then imo